content/docs/tasks/traffic-management/app-health-check/index.md

```diff
@@ -114,27 +128,111 @@ Remove the mutual TLS policy and corresponding destination rule added in the ste
 
 ## Liveness and readiness probes with HTTP request option
 
-This section shows how to configure health checking with the HTTP request option.
+This section shows how to configure health checking with the HTTP request
+option. Firstly it discusses the case when the service is using same port
+for regular traffic and health checking. Then it discusses the case with
+different port.
 
-### Mutual TLS is disabled
+### Mutual TLS is disabled (same port)
```

Comment:
This heading is not descriptive and feels incomplete. I suggest:
HTTP readiness and liveness check with mutual TLS disabled on the same port
Please also change the other three headings accordingly.